phpBB/styles/prosilver/template/overall_footer.html

```diff
@@ -108,7 +107,6 @@ <h3 class="alert_title">&nbsp;</h3><p class="alert_text"></p>
 <!-- EVENT overall_footer_after -->
 
 <!-- IF S_PLUPLOAD --><!-- INCLUDE plupload.html --><!-- ENDIF -->
-{$ICONIFY_ICONS}
```

Comment:
There was a js here related with CDN (S_ALLOW_CDN), is not needed anymore?

Reply:
do you mean this part? https://github.com/phpbb/phpbb/blob/3.3.x/phpBB/styles/prosilver/template/overall_footer.html#L71

Reply:
yes

Reply:
That was a fallback, I'll add it back as well.